documentation/marketing_usage.md

```diff
@@ -0,0 +1,112 @@
+Using the Marketing app
+-----------------------
+
+Email Marketing App/functionality allows sending bulk followup emails to `Subscribers` after a predefined number of days from their instance activation. The app allows scheduling and personalizing the emails using django templates. Some of the usage examples include:
+
+1. Sending email to all users on the 25th day of their trial to remind them of the trial ending.
+2. Sending emails to users on the 10th day of their trial explaining additional features and services offered by Opencraft.
+3. Sending a feedback form after their end of trial.
+
+## Email Templates
+
+### Creating a new followup email template
```

Comment:
@arjunsinghy96, We should mention setting up `MARKETING_EMAIL_REPORT_RECIPIENTS` and `MARKETING_EMAIL_SENDER` as well.

Reply:
@shimulch Thanks for catching this. I have added a `Configuration Settings` section to address this.